bin/MadGraph5_aMCatNLO/cards/production/13TeV/darkphotonDimuon/darkphotonDimuon_DPMass5e-1MeV_DF1e-2/darkphotonDimuon_DPMass5e-1MeV_DF1e-2_proc_card.dat

```diff
@@ -0,0 +1,10 @@
+import model --modelname  HAHM_MG5model_v3/HAHM_variableMW_v3_UFO
+
+define p = g u c d s b u~ c~ d~ s~ b~
+define j = g u c d s u~ c~ d~ s~
+define l+ = e+ m+
+define l- = e- m-
```

Comment:
why is this needed ?

Reply:
Madgraph is not happy with the HAHM model that removes mu+ and mu-, and then replaces them with m+ and m-. It requests me to redefine the l+ and l-, even though I am not using them in my model.

Reply:
@y19y19 If you remove --modelname it would be okay. Is there any reason to use this option?

Reply:
if I remove --modelname, madgraph also complains:
The import format was not given, so we guess it as command
import model HAHM_MG5model_v3/HAHM_variableMW_v3_UFO
INFO: failed to load model from pickle file. Try importing UFO from File 
Command "import /uscms_data/d3/yyao/DarkPhoton_2/Gridpack2/genproductions/bin/MadGraph5_aMCatNLO/darkphotonDimuon_DPMass5e-1MeV_DF1e-2/darkphotonDimuon_DPMass5e-1MeV_DF1e-2_gridpack/work/darkphotonDimuon_DPMass5e-1MeV_DF1e-2_proc_card.dat" interrupted in sub-command:
"define higgs = h hs" with error:
InvalidCmd : No particle hs in model

I am not sure how to make it work. Let me know if you have an idea.